Ghidra/Features/Base/src/main/java/ghidra/app/util/viewer/field/AnnotatedStringHandler.java

```diff
@@ -95,4 +95,11 @@ public boolean handleMouseClick(String[] annotationParts, Navigatable sourceNavi
 	 * @return the example of how this is used.
 	 */
 	public String getPrototypeString();
+	
+	/**
+	 * Returns an example string of how the annotation is used
+	 * @param contained The text that may be wrapped
+	 * @return the example of how this is used.
+	 */
+	public String getPrototypeString(String contained);
```

Comment:
This method should be made into default method so that any external non-Ghidra developers implementing this interface will not have a compile error.  The default implementation should simply call the original method, ignoring the parameter passed in.

```
public default String getPrototypeString(String displayText) {
    return getPrototypeString();
}
```